build/LegacyRuntimes.props

```diff
@@ -0,0 +1,57 @@
+<Project>
+  <PropertyGroup Condition=" '$(IncludeLegacySharedFramework)' == 'true' ">
+    <Legacy110CoreSetupChannel>release/1.1.0</Legacy110CoreSetupChannel>
```

Comment:
What's wrong with "Additional"?

In general, I think all the "legacy" stuff you are adding and the "additional" stuff that is currently there needs to be rectified.  We shouldn't have both.